src/eosjs2-jsonrpc.ts

```diff
@@ -135,27 +135,27 @@ export class JsonRpc implements AuthorityProvider {
 
   /** Raw call to `/v1/chain/get_abi` */
   public async get_abi(accountName: string): Promise<GetAbiResult> {
-    return await this.fetch("/v1/chain/get_abi", { accountName });
+    return await this.fetch("/v1/chain/get_abi", { account_name: accountName });
   }
 
   /** Raw call to `/v1/chain/get_account` */
   public async get_account(accountName: string): Promise<any> {
-    return await this.fetch("/v1/chain/get_account", { accountName });
+    return await this.fetch("/v1/chain/get_account", { account_name: accountName });
   }
 
   /** Raw call to `/v1/chain/get_block_header_state` */
   public async get_block_header_state(blockNumOrId: number | string): Promise<any> {
-    return await this.fetch("/v1/chain/get_block_header_state", { blockNumOrId });
+    return await this.fetch("/v1/chain/get_block_header_state", { block_num_or_id: blockNumOrId });
```

Comment:
Yikes! Someone snuck through a parameter name change in another pull?

Reply:
This should get another pull request to fix. The fix is to restore the original names; they shouldn't have changed.